packages/stencil-library/src/interface.d.ts

```diff
@@ -1 +1,3 @@
-export type Mode = 'light' | 'dark';
+export type Mode = 'light' | 'dark' | 'danger';
```

Comment:
[Dust] Why is danger a mode rather than a variant?

Reply:
This aligns with the design where there's the 3 variants (primary/secondary/link) and each has a version of each mode.